### DISCUSSION TITLE
Search for PRs without any requested reviewers

### DISCUSSION THREAD

**Discussion by michaldudak** [community/community#59377] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hi! In order to ensure that all PRs have someone assigned to them, I'd like to be able to look for the ones without any reviewers. Something like `no:review-requested` would be great.

**Comment by doncjohn** [comment]
I do not think there is currently a first-class search qualifier that means “no requested reviewers” in the same way `no:assignee` works for assignees.

The closest built-in search pieces are the positive forms, for example:

```text
is:pr is:open review-requested:USERNAME
is:pr is:open team-review-requested:ORG/TEAM
```

Those help find PRs requested from a person or team, but they do not give you the inverse across all possible requested reviewers.

For now, the most reliable workaround is to query open PRs and inspect the review request fields through the API/CLI. For example with the GitHub CLI:

```bash
gh pr list --state open --limit 100   --json number,title,url,reviewRequests   --jq '.[] | select((.reviewRequests | length) == 0) | "#(.number) (.title) (.url)"'
```

If you prefer the REST API, each pull request has requested reviewer/team information available through the review-requests endpoint, so a scheduled script or Action can flag PRs where both requested reviewer lists are empty.

One caveat: “no requested reviewers” is not always the same as “nobody is responsible.” A PR might have a required CODEOWNERS review that has not been requested yet, an assigned person, or an already-submitted review. So in automation I would usually check several signals:

- no current review requests
- no assignees, if your team uses assignees for PR ownership
- no approving/reviewing activity yet, if that matters
- not draft

As a product feature request, `no:review-requested` or `review-requested:none` would still be very useful, but today I would handle it with `gh pr list --json reviewRequests` or a small workflow.
